docs/user/images/plot_images.py

```diff
@@ -49,7 +49,16 @@ def plot_and_save_functions(module, module_name):
             ):
                 result = func()
                 if isinstance(result, tuple):
-                    result = result[0]
+                    result = [item for item in result if isinstance(item, Device)]
+                    for result in result:
```

Comment:
Consider renaming the `result` 'variable' to make it different from the `result` 'list' ?